repondeur/zam_repondeur/views/auth.py

```diff
@@ -21,7 +24,11 @@ def get(self) -> Any:
     @view_config(request_method="POST")
     def post(self) -> Any:
         email = self.request.params["email"].strip().lower()
-        headers = remember(self.request, email)
+        user, created = get_one_or_create(User, email=email)
+        user.last_login_at = datetime.utcnow()
+        if created:
```

Comment:
I wonder if we should put that flush directly in `get_one_or_create`?

Reply:
I don't think we should do that. (I'm not even sure I like doing the `add` in there...)